docs/static/plugin-generator.asciidoc

```diff
@@ -0,0 +1,19 @@
+[[plugin-generator]]
+== Generating Plugins
+
+You can now create your own Logstash plugin in seconds! The generate subcommand of `bin/logstash-plugin` creates the foundation 
```

Comment:
might be write, `You can now create your own Logstash plugins easier!`, the in a second is don't know, strange ... because end of the day they still have to do so many other things to finish the plugin, right?